src/midend/binaryAnalyses/instructionSemantics/DataFlowSemantics2.C

```diff
@@ -77,6 +77,7 @@ public:
     createOptionalMerge(const BaseSemantics::SValuePtr &other, const BaseSemantics::MergerPtr&,
                         SMTSolver*) const ROSE_OVERRIDE {
         TODO("[Robb P. Matzke 2015-08-10]");
```

Comment:
Is there some way to convince the Microsoft compiler that certain functions cannot return? Has anyone tested whether non-Microsoft compilers, when running with optimization and warnings, don't complain that the "return" is unreachable?